test/integration/consul-container/test/peering/rotate_server_and_ca_then_fail_test.go

```diff
@@ -192,8 +192,10 @@ func verifySidecarHasTwoRootCAs(t *testing.T, sidecar libservice.Service) {
 	}
 
 	retry.RunWith(failer(), t, func(r *retry.R) {
-		dump, err := libservice.GetEnvoyConfigDump(adminPort)
-		require.NoError(r, err, "could not curl envoy configuration")
+		dump, err := libservice.GetEnvoyConfigDump(adminPort, "include_eds")
+		if err != nil {
+			r.Fatal("could not curl envoy configuration")
+		}
```

Comment:
is there any reason to use `if err != nil {` rather than `equire.NoError(r, err, "could not curl envoy configuration")`